src/source-build-reference-packages/Directory.Build.targets

```diff
@@ -11,7 +11,8 @@
     contains everything. It's generated when the package is initially deconstructed and tends to be
     unique per package, so there's no benefit to this Arcade util.
   -->
-  <Target Name="InitializeStandardNuspecProperties" />
+  <Target Name="InitializeStandardNuspecProperties"
+          Condition="'$(IsPackable)' == 'true'" />
```

Comment:
I'm not in love with reusing IsPackable here...

Reply:
Agreed. These targets are only needed when using manual nuspecs in NuGet Pack instead of just a normal dotnet pack. Might be cleaner to create a property to express that and set it to true in the necessary places...

Reply:
Or you could separate this out into a separate .targets file and import it in the needed places.

Reply:
Creating a separate targets was what I was thinking as well. I think I will go that route.  Thanks for the feedback.

Reply:
Fixed with https://github.com/dotnet/dotnet/pull/1236/commits/3d934c0be0c8efaea8536b6221257ca3b1f81965